type-generator/src/main/scala/mesosphere/raml/backend/treehugger/EnumVisitor.scala

```diff
@@ -57,6 +57,20 @@ object EnumVisitor {
         VAL("DefaultValue") withType(name) := REF(underscoreToCamel(camelify(defaultValue)))
       }
     )
-    Seq(baseTrait.withDoc(comments), obj)
+
+    val jacksonSerializerSym = RootClass.newClass(name + "Serializer")
+
+
+    val jacksonSerializer = CLASSDEF(jacksonSerializerSym).withParents("com.fasterxml.jackson.databind.ser.std.StdSerializer[" + name + "](classOf[" + name + "])") := BLOCK(
```

Comment:
Could you add a docs string describing what the code would look like? We did not do this in the past but it would probably help.